spec/features/user_views_vendor_details_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+feature 'user views vendor details', %{
+  As a visitor
+  I want to view the details of a specific truck/vendor
+  So that I can learn more about the truck/vendor I’m interested in visiting
+} do
+  # Acceptance Criteria
+
+  # * As a visitor
+  # * When I visit a specific vendors page
+  # * I should be able to see the details of and reviews for said vendor
```

Comment:
See comment below on what acceptance criteria should look like - this looks more like a user story, but you only need one user story (the one you wrote in lines 3-5!)